codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java

```diff
@@ -77,7 +77,10 @@ public void addConfigInterfaceFields(
             TypeScriptWriter writer
     ) {
         if (isAwsService(settings, model)) {
-            writer.writeDocs("Unique service identifier.\n@internal")
+            ServiceShape service = settings.getService(model);
+            String sdkId = service.getTrait(ServiceTrait.class).map(ServiceTrait::getSdkId)
+                    .orElse("");
+            writer.writeDocs(String.format("Unique service identifier.%n@internal%n@default \"%s\"", sdkId))
```

Comment:
Can you run the codegen (`yarn generate-clients`) for this and commit it, or want us to do it?

Reply:
I though that was done in a separate process.

Reply:
I ran the generate-clients script now.

Reply:
i reverted the clients update in order to get this merged